aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -700,16 +684,40 @@ module aptos_token::token {
         }
     }
 
+    public fun create_royalty(royalty_points_numerator: u64, royalty_points_denominator: u64, payee_address: address): Royalty {
+        assert!(royalty_points_numerator <= royalty_points_denominator, error::invalid_argument(EINVALID_ROYALTY_NUMERATOR_DENOMINATOR));
```

Comment:
royalty_points_denominator cannot be zero, it should be an error

Reply:
great point from our storage buddy lol, will update : )